### DISCUSSION TITLE
Significant Drop in Model Performance After Adding New Samples in PaddleOCR

### DISCUSSION THREAD

**Discussion by OleksandrZhytnyk** [PaddlePaddle/PaddleOCR#13498] [Q&A]
**Description of the Problem**
Hello PaddleOCR Community,

I've encountered a significant issue with my OCR model, and I hope to get some insights and assistance from the community.

Background

In April this year, I trained an OCR model using PaddleOCR and achieved satisfactory performance metrics, with an Hmean of 0.82. The model was performing well on the validation set. 

The **data** consists of images of PDF documents that use the English language, labeled using [PPOCRLabel](https://github.com/Mushroomcat9998/PaddleOCR/blob/main/PPOCRLabel/README.md).

The **model** configuration used was [ch_PP-OCRv4_det_teacher.yml](https://github.com/PaddlePaddle/PaddleOCR/blob/main/configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_teacher.yml).

Issue
Recently, I added 800 new samples to my training dataset to further improve the model. However, after adding these new samples and retraining the model using the same config.yaml file, I've observed a strange behavior. Initially, the model's performance metrics dropped from an Hmean of 0.7 to 0.1 after just a few epochs (if i use lower lr, the metric dropped slowly). I also tested it on the April version dataset and encountered the same behavior.

Steps Taken to Fix the Issue

1. Updated the PaddleOCR repository to the latest version.
2. Checked the labels for correctness.
3. Used the default configuration file.

Observations
The model's performance degrades significantly after adding the new samples.
The drop in performance is abrupt and happens within the first few epochs.
The same config.yaml file was used for both training sessions.

Here is my config that i used

Global:
  debug: false
  use_gpu: true
  epoch_num: &epoch_num 500
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/det_PP-OCRv4
  save_epoch_step: 500000
  eval_batch_step:
  - 0
  - 83
  cal_metric_during_train: false
  checkpoints: null
  pretrained_model: ./ch_PP-OCRv4_det_server_train/best_accuracy.pdparams
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./checkpoints/det_db/predicts_db.txt
  distributed: true

Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: PPHGNet_small
    det: True
  Neck:
    name: LKPAN
    out_channels: 256
    intracl: true
  Head:
    name: PFHeadLocal
    k: 50
    mode: "large"
    

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: RMSProp
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0001 #(8*8c)
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 1e-6

PostProcess:
  name: DBPostProcess
  thresh: 0.4
  box_thresh: 0.7
  max_candidates: 4000
  unclip_ratio: 2

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./det/dataset_det
    label_file_list:
      - ./det/dataset_det/train.txt
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - CopyPaste: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 640
        - 640
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: *epoch_num
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: *epoch_num
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 12
    num_workers: 12

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./det/dataset_det/
    label_file_list:
      - ./det/dataset_det/test.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest: null
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 12
profiler_options: null

![photo_2024-07-25_11-36-59](https://github.com/user-attachments/assets/0bfea764-07ed-4c41-9c99-4611b92d35e7)

I would appreciate any help or advice that can explain this behavior. Please feel free to ask questions if you need further clarification about my case.

**Comment by MykytaKyt** [comment]
I had the same problem. Does anyone solve this?

**Comment by SWHL** [comment]
Duplicated with #13540
